kzz7yh-a56778/cod-ve07v1_kzz7yh-a56778.xml

```diff
@@ -190,10 +191,10 @@
    </p>
                <p xml:id="kzz7yh-a56778-d1e338">
                   <lb ed="#V" n="120"/>Ad primum in oppositum cum dicitur quod pater et filius et 
-      spi<lb ed="#V" n="121" break="no"/>sanctus sunt vnus qua creat etc. Dico quod 
+      spiritus<lb ed="#V" n="121" break="no"/>sanctus sunt vnus qua creat etc. Dico quod 
```

Comment:
I know the type face seem like it treats this as one word, but for the sake of uniformity, I'd would like to recognize it has two. So we should always have a break between "spiritus sanctus"

```suggestion
      spiritus
      <lb ed="#V" n="121"/>sanctus sunt vnus qua creat etc. Dico quod 
```